lib/db.js

```diff
@@ -3,7 +3,11 @@ export const projects = [
     slug: "1",
     id: 0,
     title: "project 1",
-    tasks: [{ task: "task 1" }, { task: "task 2" }, { task: "task 3" }],
+    tasks: [
+      { task: "task 1", isDone: false, id: "crypto here" },
+      { task: "task 2", isDone: false, id: "crypto here" },
+      { task: "task 3", isDone: false, id: "crypto here" },
```

Comment:
your id's are not unique here.

Reply:
Forgot to delete the db.js 😅 There's no longer need for it.